tekton/src/pipelines/gitops/deprovision-mas-instance.yml.j2

```diff
@@ -186,3 +192,91 @@ spec:
       taskRef:
         kind: Task
         name: gitops-process-mongo-user
+{% if deprovision_manage_addons == true %}
+    - name: gitops-deprovision-efs
+      runAfter:
+        - gitops-process-mongo-user
+      params:
+        - name: cluster_name
+          value: $(params.cluster_name)
+        - name: avp_aws_secret_region
+          value: $(params.avp_aws_secret_region)
+        - name: avp_aws_secret_key
+          value: $(params.avp_aws_secret_key)
+        - name: avp_aws_access_key
+          value: $(params.avp_aws_access_key)
+        - name: mas_instance_id
+          value: $(params.mas_instance_id)
+        - name: cloud_provider
+          value: $(params.cloud_provider)
+      workspaces:
+        - name: configs
+          workspace: configs
+      taskRef:
+        kind: Task
+        name: gitops-deprovision-efs
+    - name: gitops-delete-kafka-config
```

Comment:
I believe for the deprovision of kafka-config we should just call it anyway regardless as if there is no config then it shouldn't cause a problem. So we should remove the deprovision_manage_addons check in the templating.

Reply:
removed the deprovision_manage_addons from deprovision-mas-instance.yml.j2 and  generate-tekton-pipelines.yml